controller/src/main/java/io/pravega/controller/store/stream/ZKStreamMetadataStore.java

```diff
@@ -174,20 +175,30 @@ ZKScope newScope(final String scopeName) {
     }
 
     @Override
-    public CompletableFuture<Pair<List<String>, String>> listScopes(String continuationToken, int limit, 
+    public CompletableFuture<Pair<List<String>, String>> listScopes(String continuationToken, int limit,
                                                                     Executor executor, long requestId) {
         // Pagination not supported for zk based store. 
         throw new UnsupportedOperationException();
     }
 
     @Override
-    public CompletableFuture<Void> addReaderGroupToScope(String scopeName, String rgName, UUID readerGroupId, 
+    public CompletableFuture<Void> addStreamTagsToIndex(String scope, String name, StreamConfiguration config, OperationContext context, Executor executor) {
+        return CompletableFuture.completedFuture(null);
```

Comment:
unspported

Reply:
throwing unsupported causes in memory tests to fail, hence returning a completed future. Let me know if this should be modified.